README.md

```diff
@@ -7,6 +7,19 @@
 [![Discord][discord-shield]][discord-url]
 
 
+# Kontent.ai React Integration Template
+
+This template can be used to jumpstart your own integration development with Kontent.ai. It contains all the neccessary tools for creating a new [Custom Element](https://kontent.ai/learn/tutorials/develop-apps/integrate/content-editing-extensions/), a UI extension for content editors. 
+
+
+You can inspire yourself by browsing already created integrations [**here**](https://github.io/topics/kontent-ai-integration).
+
+If you wish to include your integration into the mentioned list, please add the **kontent-ai-integration** topic into your github integration repository. 
+
+Additional information and tutorials can be found on [kontent-ai.github.io](https://kontent-ai.github.io/).
```

Comment:
Great - we might introduce another badge for thet - what do you thing.

And we need to sync about the purpose of the kontent-ai.github.io